plugins/imagespacingbox/plugin.js

```diff
@@ -14,23 +14,65 @@
 						return;
 					}
 
+					var getFocusedWidget = function (editor) {
+						var widget = editor.widgets.focused;
+
+						if (widget && widget.name == 'image') return widget;
+
+						return null;
+					};
+
+					var getImageHVSpace = function (widget) {
+						var imageHVSpace = {hspace: '', vspace: ''};
```

Comment:
Why are you making this an "internal" function, couldn't this be declared directly on the plugin?